### DISCUSSION TITLE
Onion Chart,  is this possible?

### DISCUSSION THREAD

**Discussion by manulsan** [graphieros/vue-data-ui#43] [Q&A] [answered]
While using Onion chart,
if no  series name, can you remove ":"  also?
Just display value or percent.

- Picture
![image](https://github.com/graphieros/vue-data-ui/assets/31757599/9d085c47-ec84-4e3c-8766-bcd306a1e0e6)

- We are applying as below
![image](https://github.com/graphieros/vue-data-ui/assets/31757599/cb31f614-4b5b-4440-bfaa-a03e4d54e25a)

Thanks

**Comment by graphieros** [comment, marked as answer]
You can upgrade to [v2.2.19](https://github.com/graphieros/vue-data-ui/releases/tag/v2.2.19)

**Reply by manulsan** [reply]
^^
I will try it.

**Comment by manulsan** [comment]
Question> for background color
please refer below picture,

![image](https://github.com/graphieros/vue-data-ui/assets/31757599/d644ebb6-2bd1-47e7-8d5a-956fc9f11d6f)

Thanks

**Reply by graphieros** [reply]
I'm not sure I understand your question.

If your problem is that background color is different than your container background color when using a theme, then you can apply the same background color to your container.

You can also choose to not use a theme, and configure the background color to be the same as your container.
